spec/requests/users_request_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
+      expect(response.body).to include('The list of users')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/1'
+      expect(response).to have_http_status(:success)
+      expect(response.status).to eq(200)
+      expect(response).to render_template('show')
+      expect(response.body).to include('User name with posts')
+    end
```

Comment:
- [ ] **[OPTIONAL]** I think it would be more descriptive to divide these tests.

Reply:
Will Do 👍